pkg/core/resources/manager/customizable_manager.go

```diff
@@ -35,6 +38,15 @@ func (m *customizableResourceManager) Delete(ctx context.Context, resource model
 	return m.resourceManager(resource.GetType()).Delete(ctx, resource, fs...)
 }
 
+func (m *customizableResourceManager) DeleteAll(ctx context.Context, fs ...store.DeleteAllOptionsFunc) error {
+	for _, typ := range m.registry.ListTypes() {
```

Comment:
Similarly to the comment in `ResourceManager`, I think, business logic that goes through the list of all supported types and calls `DeleteAll` on each if them is specific to `Mesh`.

`customizableResourceManager` is just a dispatcher. I would expect `DeleteAll` to be dispatched to a single type-specific `ResourceManager` (like it happens for other `CRUDL` operations)